### ISSUE TITLE
Body parts missing and hair stiff

### ISSUE THREAD

**Issue by dawnandstars** [xivdev/Penumbra#300]
When disabling mods, anything that is modded suddenly no longer shows in game, resulting in missing/invisible body parts and hairs to be completely stiff and unmoving when running around and such. This hasn't happened before, but since the update it is happening every time I disable mods

**Comment by Theo-Asterio** [comment]
Are you on the most recent version of penumbra 0.7.0.3 and have you restarted since updating?

**Comment by dawnandstars** [comment]
Yes I am on the most recent version and I have restarted since the update, the invisible parts and stiff hair still happens

**Comment by Ottermandias** [comment]
Metadata edits currently still apply even with mods disabled, so anything with EQDP edits will cause parts to be missing and anything with EST edits will cause bones to be wrong etc.
Until it is fixed, you can use a "Use No Mods" assignment on the relevant characters instead of disabling mods, or just don't disable them.
I know what causes the issue, just not how I want to handle it yet.

**Comment by dawnandstars** [comment]
Thank you so much, I'll be sure to do that!
